src/interfaces/uproxy_core_api.ts

```diff
@@ -290,5 +302,12 @@ export interface CoreApi {
   // This may also invoke an OAuth flow, in order to perform operations
   // with the cloud computing provider on the user's behalf.
   cloudInstall(args:CloudInstallArgs): Promise<void>;
+
+  // Invokes OAuth flow and destroys uProxy cloud server on the 
+  // provider specified.
+  cloudDestroy(providerName :string) : Promise<void>;
```

Comment:
I could go either way here.  I agree there is a bit too much boiler plate here, but overloading ```CloudOperationArgs``` could get weird as each command would require different args (install requires the region, but stop doesn't, I assume ```cloudResize``` would require the new size, etc).  Is there some way we can type ```CloudOperationArgs``` so that the compiler can check that we are passing the right params for that action type?